seed/serializers/rules.py

```diff
@@ -0,0 +1,105 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+
+from rest_framework import serializers
+from seed.models.data_quality import Rule
+from seed.models import StatusLabel
+
+
+class RuleSerializer(serializers.ModelSerializer):
+    data_type = serializers.CharField(source='get_data_type_display', required=False)
+    status_label = serializers.PrimaryKeyRelatedField(
+        queryset=StatusLabel.objects.all(),
+        allow_null=True,
+        required=False
+    )
+    severity = serializers.CharField(source='get_severity_display', required=False)
+
+    class Meta:
+        model = Rule
+        fields = [
+            'condition',
+            'data_type',
+            'enabled',
+            'field',
+            'id',
+            'max',
+            'min',
+            'not_null',
+            'required',
+            'rule_type',
+            'severity',
+            'status_label',
+            'table_name',
+            'text_match',
+            'units',
+        ]
+
+    def validate_label(self, label):
+        """
+        Note: DQ Rules can be shared from parent to child but child orgs can
+        have their own labels. So, a Rule shouldn't be associated to Labels
+        from child orgs. In other words, Rule and associated Label should be
+        from the same org.
+        """
+        if label is not None and label.super_organization_id != self.instance.data_quality_check.organization_id:
+            raise serializers.ValidationError(
+                f'Label with ID {label.id} not found in organization, {self.instance.data_quality_check.organization.name}.'
+            )
+        else:
+            return label
+
+    def validate(self, data):
+        """
+        These are validations that involve values between multiple fields.
+
+        Custom validations for field values in isolation should still be
+        contained in 'validate_{field_name}' methods which are only checked when
+        that field is in 'data'.
+        """
+        data_invalid = False
+        validation_messages = []
+
+        # Rule with SEVERITY setting of "valid" should have a Label.
+        severity_is_valid = self.instance.severity == Rule.SEVERITY_VALID
+        severity_unchanged = 'get_severity_display' not in data
+        severity_will_be_valid = data.get('get_severity_display') == "valid"
```

Comment:
Not sure if this is any better, but maybe we could do `dict([(display, val) for val, display in Rule.SEVERITY])[Rule.SEVERITY_VALID]` instead of comparing directly to a string literal